src/montgomerybuilder.js

```diff
@@ -311,6 +313,154 @@ function buildFromMontgomery(fn, q) {
     });
 }
 
+// no adx assembler
+
```

Comment:
How about removing the empty line here?

Reply:
removed // no adx assembler  a leftover , blanks condensed